### PR TITLE
remove rhoda and rhods from performance tests

### PR DESCRIPTION
**Pull request by mfrancisc**
operators were uninstalled from sandbox , removing those also from the performance tests suite

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Comment by mfrancisc** [comment]
I'm only unsure about this comment https://github.com/codeready-toolchain/toolchain-e2e/blob/a0d6f321a3c1d9c6ccff93f89ded49ba4e8333a5/setup/operators/operators.go#L33 ?

Should that be renabled now ?

**Comment by alexeykazakov** [comment]
@rajivnathan please take a look :)

**Comment by MatousJobanek** [comment]
@alexeykazakov @mfrancisc 
I'm sorry, but why do we remove RHODS? This operator (add-on) is still part of DevSandbox,right?

**Comment by mfrancisc** [comment]
Did I misunderstood which operators we are removing ? I saw in sandbox-sre pr we rmoved rhods https://github.com/codeready-toolchain/sandbox-sre/pull/949

but maybe I'm misunderstood!

File: setup/operators/operators.go [hunk 1]
```diff
@@ -24,13 +24,11 @@ const (
 
 var Templates = []string{
 	"devspaces.yaml",
-	"rhoda.yaml",
-	"rhods.yaml",
```
Comment (rajivnathan) [inline setup/operators/operators.go:28]: Reminder to keep rhods as discussed in meeting

**Review by rajivnathan — CHANGES_REQUESTED** [review]

File: setup/operators/operators.go (hunk 1 above)
Comment (mfrancisc) [reply setup/operators/operators.go:28]: done see [d8724f9](https://github.com/codeready-toolchain/toolchain-e2e/pull/675/commits/d8724f9b0bae5839e89d1a07880ebaf3939bf759) 

Sorry for the confusion!

Comment (rajivnathan) [reply setup/operators/operators.go:28]: the rhods template file needs to be restored as well

Comment (mfrancisc) [reply setup/operators/operators.go:28]: Good point! 🤦  

Restored in [f022835](https://github.com/codeready-toolchain/toolchain-e2e/pull/675/commits/f022835dee599e36bcb642f96b8ff69428ffe463)

Thanks for pointing that out!

**Review by alexeykazakov — APPROVED** [review]
Yes! My bad. I missed the rhods part when did the review last time. We do want to keep RHODS! Now all looks good!

**Review by xcoulon — APPROVED** [review]

**Pull request merged**